unable to build Android app

while trying to build android app with react native status keycard , i am getting following errors 

> /react/PackageList.java:60: error: cannot find symbol
import com.reactlibrary.RNStatusKeycardPackage;
                       ^
  symbol:   class RNStatusKeycardPackage
  location: package com.reactlibrary
/android/app/build/generated/rncli/src/main/java/com/facebook/react/PackageList.java:131: error: cannot find symbol
      new RNStatusKeycardPackage(),
          ^
  symbol:   class RNStatusKeycardPackage
  location: class PackageList
2 errors

FAILURE: Build failed with an exception.

However changing package import manually works but it shouldn't be like this. Ideally it should generate and  link packages properly . I have followed all the steps mentioned in the page documentation but still the error doesnt seem to go away.

--- a/android/src/main/AndroidManifest.xml
+++ b/android/src/main/AndroidManifest.xml
@@ -1,6 +1,6 @@
 
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.reactlibrary">
+          package="im.status.ethereum.keycard">
 
 </manifest>
   

